scopes/component/ui/version-dropdown/version-info/version-info.tsx

```diff
@@ -46,16 +55,20 @@ export function VersionInfo({
 
   const href = overrideVersionHref ? overrideVersionHref(version) : `?version=${version}`;
 
+  const formattedVersion = useMemo(() => {
+    return tag ? version : version.slice(0, 6);
+  }, [tag, version]);
+
+  const isLatest = version === latestVersion;
+
   return (
-    <div ref={currentVersionRef}>
+    <div ref={ref || currentVersionRef}>
       <MenuLinkItem active={isCurrent} href={href} className={styles.versionRow}>
         <div className={styles.version}>
           <UserAvatar size={24} account={author} className={styles.versionUserAvatar} showTooltip={true} />
-          <Ellipsis className={classNames(styles.versionName, tag && styles.tag, !tag && styles.snap)}>
-            {version}
-          </Ellipsis>
-          {version === latestVersion && <VersionLabel className={styles.label} status="latest" />}
-          {showDetails && commitMessage(message)}
+          <Ellipsis className={classNames(styles.versionName)}>{formattedVersion}</Ellipsis>
+          {isLatest && <VersionLabel status="latest" />}
+          <CommitMessage message={message} showDetails={showDetails} />
         </div>
         <Ellipsis className={styles.versionTimestamp}>
```

Comment:
Same here, you can remove the ellipsis style from the class, it’s already exist in the `Ellipsis` component.